LICENSE.md

```diff
@@ -26,13 +30,16 @@ Version 2.0.3
 
 **Modpack Creator**: An individual who creates modpacks.
 
-**Author(s)**: Original programmer or programmers responsible for writing the Mod.
+**Author(s)**: Original programmer(s), texturer(s), and contributors responsible for creating, in part or in whole, the Mod.
 
 **Contributor**: Programmer or programmers who contribute code to the Mod, but are not the Author.
 
 **Codebase**: The Mod source code, complete with source history and Contributor records.
 
-**Distributor**: Anyone who distributes the compiled mod.
+**Distributor**: Anyone who distributes the mod, in source or object form.
```

Comment:
I intentionally differentiated between the source form and the compiled form in the definition of "distributor", because technically using git as a version control system is distributing the source code. Also, I have no problem with someone handing my code to their friends...this license is (in part) intended to prevent people from re-hosting the compiled binaries and making money from it.